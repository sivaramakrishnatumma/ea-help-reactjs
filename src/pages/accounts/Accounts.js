import React from "react";

const Accounts = () => {
  return (
    <div className="page-container">
      <h1 className="center">Accounts Page</h1>
    </div>
  );
};

export default Accounts;
