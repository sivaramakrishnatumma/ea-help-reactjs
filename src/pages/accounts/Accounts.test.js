import React from "react";
import { shallow } from "enzyme";
import Accounts from "./Accounts";

describe("Accounts page test case", () => {
  it("should test Accounts Page", () => {
    const wrapper = shallow(<Accounts />);
    expect(wrapper).toMatchSnapshot();
  });
});
