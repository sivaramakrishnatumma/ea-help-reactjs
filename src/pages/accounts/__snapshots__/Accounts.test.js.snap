// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accounts page test case should test Accounts Page 1`] = `
<div
  className="page-container"
>
  <h1
    className="center"
  >
    Accounts Page
  </h1>
</div>
`;
